ping/_compositions/go-rust-interop-latest.toml

```diff
@@ -90,11 +90,9 @@
         path = "./rust/"
 
       [groups.build_config.build_args]
-        CARGO_FEATURES = '{{ .CargoFeatures }}'
-        CARGO_REMOVE = '{{ .CargoFeatures }}'
-        CARGO_PATCH = """
-          {{ .CargoFeatures }} = {package = "libp2p", git = "https://{{ or $.Env.GitTarget "github.com/libp2p/rust-libp2p" }}", rev = "{{ $.Env.GitReference }}", default_features = false, features = [ "websocket", "mplex", "yamux", "tcp", "ping", "noise", "dns", "async-std", "rsa" ], optional = true}
-          """
+        BINARY_NAME = '{{ .BinaryName }}'
+        GIT_TARGET = '$.Env.GitTarget'
+        UPDATE_CMD = '{{ .UpdateCmd }} --precise {{ $.Env.GitReference }}'
```

Comment:
:tada:

Reply:
Unfortunately, this doesn't seem to work for Git dependencies that point to something else other than the master branch.